src/SFA.DAS.AssessorService.Database/PostDeploymentScripts/EPAO_MIgration_Part4_Apply.sql

```diff
@@ -0,0 +1,18 @@
+﻿-- Migrate Apply table - set the Standardreference
```

Comment:
I'm a nitpicker, can we rename the file on this to lose the capital I in Migration?

Reply:
Yep. That is nitpicking. Didn't even spot it.

Reply:
It seems it is case insensitive -so change of filename is not happening.